### PR TITLE
Adds two alternate shuttles

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
rscadd: Nanotrasen is short on cash, and are now borrowing escape shuttles from other stations, and Space Apartments.
/:cl:
wnqbuld
qnr1hwn

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Bloat

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -112,6 +112,24 @@
 		without protective goggles. Emitters spawn powered on, expect \
 		admin notices, they are harmless."
 
+/datum/map_template/shuttle/emergency/imfedupwiththisworld
+	suffix = "imfedupwiththisworld"
+	name = "Oh, Hi Mark"
+	description = "How was work today? \
```
Comment (KorPhaeron) [inline code/datums/shuttles.dm]: copyrighted material

**Comment by KorPhaeron** [comment]
I'll merge it when you remove the stolen movie script

**Comment by Iamgoofball** [comment]
there I fixed it Kor, 100% original script

File: code/datums/shuttles.dm [hunk 2]
```diff
@@ -112,6 +112,24 @@
...
 
+/datum/map_template/shuttle/emergency/imfedupwiththisworld
+	suffix = "imfedupwiththisworld"
+	name = "Oh, Hi Daniel"
+	description = "How was space work today? \
+		Oh, pretty good. We got a new space station and the company will make a lot of money. \
+		What space station? \
+		I cannot tell you; it's space confidential. \
+		Aw, come space on. Why not? \
+		No, I can't. Anyway, how is your space sex life?"
```
Comment (PKPenguin321) [inline code/datums/shuttles.dm]: erp is banned sorry

**Comment by coiax** [comment]
As head of alternative shuttles, I'm afraid I cannot allow such forced references to other servers and movies in the game.

Better luck next time.

**Comment by pudl** [comment]
shut the fuck up @coiax

**Comment by Iamgoofball** [comment]
I HAVE BEEN OUTMEMED

**Pull request merged**